src/parser.ts

```diff
@@ -45,7 +45,7 @@ export function parse(
     return parseLiteral(schema, keyName)
   }
 
-  const types = typesOfSchema(schema)
+  const types = typesOfSchema(schema, options)
```

Comment:
```suggestion
  const types = typesOfSchema(schema)
```

Reply:
options needs to be passed in so that the matchers can read the `enableBigInt`  option